diff -Nru giflib-5.2.1-old/Makefile giflib-5.2.1-new/Makefile
--- giflib-5.2.1-old/Makefile	2019-06-25 01:08:57.000000000 +0900
+++ giflib-5.2.1-new/Makefile	2021-05-10 09:30:17.013737600 +0900
@@ -6,6 +6,7 @@
 # of code space in the shared library.
 
 #
+CC = gcc
 OFLAGS = -O0 -g
 OFLAGS  = -O2
 CFLAGS  = -std=gnu99 -fPIC -Wall -Wno-format-truncation $(OFLAGS)
@@ -22,7 +23,7 @@
 
 # No user-serviceable parts below this line
 
-VERSION:=$(shell ./getversion)
+VERSION:=5.2.1
 LIBMAJOR=7
 LIBMINOR=2
 LIBPOINT=0
@@ -33,7 +34,7 @@
 HEADERS = gif_hash.h  gif_lib.h  gif_lib_private.h
 OBJECTS = $(SOURCES:.c=.o)
 
-USOURCES = qprintf.c quantize.c getarg.c 
+USOURCES = qprintf.c quantize.c getarg.c gif_err.c
 UHEADERS = getarg.h
 UOBJECTS = $(USOURCES:.c=.o)
 
